.github/workflows/publish-dist.yml

```diff
@@ -29,9 +28,17 @@ jobs:
       - name: Commit to dist repo
         run: |
           set -ex
+          mkdir dist-repo
           cd dist-repo
+          TAR="$(ls ../cockpit-*.tar.xz)"
+          if [ $(echo "$TAR" | wc -l) -ne 1 ]; then
```

Comment:
Does this do what you think it does?  I read that as printing the list of tarballs on one line and then asking how many lines it is.

github workflows run with bash: why not just do the glob inside an array assignment and see what you get from it?

```
var=(*glob*)
echo "${#var[@]}"
```

Alternatively: is there some way we can get ourselves a bit more involved in the artifact unpack logic?

Reply:
One more concern while we're at it: we unpack the tarball from the zip file, openly and then create a subdirectory named `dist-repo` and do work inside of that.  Could the `.zip` file itself contain a `dist-repo/` subdirectory filled with extra goodies to trip us up  (like an initialised hostile git repo, for example?).  You give a path on creation, but not on unpacking...

Reply:
The .zip can contain any crap that it wants. If it contains a dist-repo, then the `mkdir dist-repo` will fail. If the built .tar contains dist-repo/, then we'll end up having dist-repo/dist-repo/, which will just be ignored.

Reply:
I also changed the workflow to unpack it into a "dist-unpack/" dir, and only grab the tarball out of it.